src/app/lists/ListDataview.tsx

```diff
@@ -214,6 +210,11 @@ const ListsDataView: React.FC<ListDataViewProps> = ({
                 },
             } as const satisfies Partial<AuditLog>;
 
+            const { error } = await supabase
+                .from("lists")
+                .delete()
+                .eq("primary_key", itemToDelete.primaryKey);
```

Comment:
Moved this to avoid audit log calling itemToDelete.itemName after the item has been deleted

Reply:
I"m not sure if this was causing any problems... but this new version is also fine? haha

Reply:
This works now 👀